lib/ex_debug_toolbar/plug/ignore_path_match.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ExDebugToolbar.Plug.IgnorePathMatch do
+  @moduledoc false
+  @behaviour Plug
+
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(conn, opts \\ []) do
+    default_paths = Keyword.get(opts, :ignore_paths, [])
+    conn |> put_private(:toolbar_ignore_path?, ignore?(conn, default_paths))
```

Comment:
should we name it `ex_debug_toolbar` to keep same name? or `debug_toolbar` at least?